tap_zoom/client.py

```diff
@@ -128,6 +138,11 @@ def request(self,
             LOGGER.warn('Rate limit hit - 429')
             raise Server429Error(response.text)
 
+        if response.status_code != 200:
+            zoom_response = response.json()
+            zoom_response.update({'status': response.status_code})
+            raise Exception('Unable to authenticate (zoom response: `{}`)'.format(zoom_response))
```

Comment:
Why do we always say `Unable to authenticate` in the logger, there must be custom error depending on `response.status_code`